### PR TITLE
Couple things

### PR DESCRIPTION
**Pull request by jdent02**

### CODE REVIEW

**Comment by jdent02** [comment]
Note: GPU denoising doesn't work with internal rendering.  Not sure why but blender just freezes up.  That's why the option is only for external render.

**Comment by bsavery** [comment]
Hmmm.  I'd have to think why...

File: presets/properties.py [hunk 1]
```diff
@@ -46,7 +46,7 @@ def get_enum_items(self, context):
 
     @classmethod
     def get_from_path(cls, lib_path):
-        group_path,preset = lib_path.rsplit('/', 1)
+        group_path, preset = os.path.split(lib_path)
```
Comment (bsavery) [inline presets/properties.py]: I actually think this doesn't give you what you want right?  maybe you'd do

tokens = os.path.split(lib_path)
preset = tokens[-1]
group_path = os.path.join(tokens[:-1])

**Comment by bsavery** [comment]
So now that the presets bit is gone for now you'll have to pull that part

**Comment by jdent02** [comment]
Should be okay to go now.

**Pull request merged**